integration-tests/src/test/groovy/com/okta/sdk/tests/it/ApplicationsIT.groovy

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2017 Okta
+ * Copyright 2020-Present Okta, Inc.
```

Comment:
nit: the original date of the copyright is actually the important one ;)

Reply:
also, @jeyadevvishnu, we can tweak it before merging, so don't worry too much about it 

Thanks again!

Reply:
Thank you @bdemers, still not clear of what should be in the header, so I will leave it up to you guys. :)

Reply:
We are on it, I know there was a little confusion, with the header formats (which was my fault)

Reply:
```suggestion
 * Copyright 2017-Present Okta, Inc.
```